src/main/java/gregtech/common/covers/CoverFluidRegulator.java

```diff
@@ -37,6 +39,12 @@
     public CoverFluidRegulator(ICoverable coverHolder, EnumFacing attachedSide, int tier, int mbPerTick) {
         super(coverHolder, attachedSide, tier, mbPerTick);
         this.transferMode = TransferMode.TRANSFER_ANY;
+        this.fluidFilter = new FluidFilterContainer(this, this::shouldShowTip, maxFluidTransferRate * 100);
+    }
+
+    @Override
+    protected boolean shouldShowTip() {
+        return transferMode != TransferMode.TRANSFER_ANY;
```

Comment:
This Tio needs to be updated to state that holding control and shift changes the increment amount by 1000mb. I could not find where this top was defined, so I am putting it here

Reply:
the tooltip is shown on `drawInForeground` on `PhantomFluidWidget`